Adding navigator layer export

bfuzzy1 opened the pull request:
🥳

privateducky reviewed (changes requested):
Great stuff!

Couple of things to update:

1. Remove any comments and unused variables
2. The template.json won't naturally get pulled down with the plugin from the app, so we'll have to load it elsewhere. Does this file exist online somewhere else? If so, we can load that way. Alternatively, we can keep it in this directory but load it from GitHub/URL vs the local directory (line 27). 

If you can get these 2 things in, I'll give this a good test.

bfuzzy1 commented:
> Great stuff!
> 
> Couple of things to update:
> 
> 1. Remove any comments and unused variables
> 2. The template.json won't naturally get pulled down with the plugin from the app, so we'll have to load it elsewhere. Does this file exist online somewhere else? If so, we can load that way. Alternatively, we can keep it in this directory but load it from GitHub/URL vs the local directory (line 27).
> 
> If you can get these 2 things in, I'll give this a good test.

I updated to grab the file remotely and cleaned up a bit. For testing purposes Line 18 is set to my repo this should be changed  once testing is complete.

bfuzzy1 commented:
Almost forgot about sub technique handling. Current solution is to remove the "." and everything after it in the technique ID until I can figure out a good way to include sub techniques on their own.
